components/MyTestimonies/MyTestimonies.js

```diff
@@ -1,21 +1,22 @@
 import React from "react";
-import { testimonies } from "../MockTestimonies"
+// import { testimonies } from "../MockTestimonies"
 import { Table, Container } from 'react-bootstrap'
 import ExpandTestimony from "../ExpandTestimony/ExpandTestimony";
 import { useAuth } from "../../components/auth"
-import { useBillContent } from "../db";
+import { useBillContent, useBills, usePublishedTestimonyListing } from "../db";
 
 const TestimonyRow = ({testimony}) => { 
   const {bill, loading} = useBillContent(testimony.billNumber)
```

Comment:
let's update this to use useBill. You can pass then pass `bill.content` to ExpandTestimony